.hermes/conveyor/work-cac2f34f/adr-001-cli-error-display-format.md

```
@@ -0,0 +1,94 @@
+```rust
+eprintln!("diffguard: catastrophic failure: {err}");
+```
+
+---
+
+## Decision
+
+Change line 642 from debug format to display format:
+
+**Before:**
+```rust
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)